app/views/content/train-to-be-a-teacher/if-you-have-experience.md

```diff
@@ -0,0 +1,20 @@
+---
+title: If you have classroom experience
+related_content:
+    Career progression: "/my-story-into-teaching/career-progression"
+    Salaries and benefits : "/salaries-and-benefits"
+    Online Q&A sessions : "/event-categories/online-q-as"
+---
+
+If you have been working in a classroom and you already have a degree, you may be able to get [qualified teacher status](/what-is-qts) (QTS) through a 12-week ‘assessment only’ programme.
+
+You’ll need to show that you already meet the standards for QTS. For example, you may meet the standards if:
+
+- you’ve already been employed as an unqualified teacher or teaching assistant for a significant amount of time (for example, 2 years) and you’ve worked in more than one school
+- you’re a [qualified teacher who trained overseas](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk)
+
+Entry criteria may vary by teacher training provider. You should ask them about eligibility before you apply.
+
+Find out more about [assessment only teacher training providers](/assessment-only-providers).
+
+We have information for people who already have QTS and would like to [return to teaching](/returning-to-teaching).
```

Comment:
If you already have QTS, find out how you can return to teaching ?

(ie 'you' not 'people'?)